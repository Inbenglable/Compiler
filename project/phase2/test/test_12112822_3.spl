struct A{
    struct B{
        struct C{
            struct D{
                int e;
            }d;
        }c;
    }b;
}a;

int main(){
    a.b.c.d.e = 1;
    a.e = 1;
    a.b.c = 1;
    return 0;
}