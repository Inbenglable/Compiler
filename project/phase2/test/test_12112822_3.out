Error type 14 at line 13: accessing an undefined structure member
Error type 5 at line 14: unmatching type on both sides of assignment
